cdap-ui/app/features/workflows/controllers/tabs/runs-ctrl.js

```diff
@@ -112,19 +112,31 @@ class WorkflowsRunsController {
   stop() {
     this.runStatus = 'STOPPING';
     this.myWorkFlowApi
-     .stopRun(params, {});
+     .stopRun(params, {})
+     .$promise
+     .then(function() {
+       this.$state.go(this.$state.current, this.$state.params, {reload: true});
```

Comment:
I am sorry @astewes I knew when we added this code but I totally forgot. We are in workflows which is already written in ES6. Could you change the 

```
function() { 
  ... 
}.bind(this)
```
syntax to arrow function?

```
() => { ... }
```

In all the places.